zeebe/expression-language/src/main/java/io/camunda/zeebe/el/impl/FeelExpressionLanguage.java

```diff
@@ -90,7 +112,11 @@ public EvaluationResult evaluateExpression(
   }
 
   private Expression parseFeelExpression(final String expression) {
+    final long startNanos = System.nanoTime();
     final Either<Failure, ParsedExpression> parseResult = feelEngine.parseExpression(expression);
```

Comment:
🔧 no `try/catch/finally`? Would be good to have one just in case.

We might also monitor the number of failures